setup.py

```diff
@@ -120,7 +120,7 @@ def find_version(*file_paths):
         ],
     },
     install_requires=[
-        'bitstring>=3.1.6,<4',
+        'bitstring>=3.1.6',
```

Comment:
Based on my above comment, maybe the following would be the cleanest and self-explaining way to avoid issues with some old `pip` versions:
```python
'bitstring>=3.1.6' if sys.version_info[:2] >= (3, 7) else 'bitstring>=3.1.6,<4',
```

(up for discussion)

Reply:
I'm giving environment markers a try:

```
'bitstring>=3.1.6; python_version>="3.7"',
'bitstring>=3.1.6,<4; python_version<"3.7"',
```

Upon reflection, I do agree that it's important to keep using the old bitstream version on unsupported Python versions, as it can be a big headache to maintain otherwise.